opam-lib failure

This has been failing for quite some time here: 

```
#=== ERROR while installing opam-lib.1.2.2 ====================================#
# opam-version 1.2.2
# os           darwin
# command      make
# path         /Users/dbuenzli/.opam/4.02.3/build/opam-lib.1.2.2
# compiler     4.02.3
# exit-code    2
# env-file     /Users/dbuenzli/.opam/4.02.3/build/opam-lib.1.2.2/opam-lib-42662-ebe0e0.env
# stdout-file  /Users/dbuenzli/.opam/4.02.3/build/opam-lib.1.2.2/opam-lib-42662-ebe0e0.out
# stderr-file  /Users/dbuenzli/.opam/4.02.3/build/opam-lib.1.2.2/opam-lib-42662-ebe0e0.err
### stdout ###
# [...]
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamJson.mli
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamJson.ml
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamLineLexer.mli
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamLineLexer.ml
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamMisc.mli
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamMisc.ml
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamScript.mli
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamScript.ml
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamVersion.mli
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,jsonm -c -g -w +a-4-9-32-41-44-45-48 -bin-annot -short-paths -safe-string -I core -I solver -I repositories -I client -I core/ core/opamVersion.ml
### stderr ###
# [...]
# File "core/opamMisc.mli", line 46, characters 26-53:
# Warning 50: ambiguous documentation comment
# File "core/opamVersion.ml", line 30, characters 2-24:
# Error: Unbound module Debian.Version
# make[5]: *** [core/opamVersion.cmo] Error 2
# make[4]: *** [byte-code-library] Error 2
# make[3]: *** [subproj_core] Error 2
# make[2]: *** [opam-core.cma] Error 2
# make[1]: *** [opam-lib.byte] Error 2
# make: *** [opam-lib] Error 2
```

--- a/packages/opam-lib/opam-lib.1.2.2/opam
+++ b/packages/opam-lib/opam-lib.1.2.2/opam
@@ -21,7 +21,7 @@ build: [
 depends: [
   "ocamlgraph"
   "cmdliner"
-  "dose" {>= "3.2.2+opam" & < "4"}
+  "dose" {>= "3.2.2+opam" & < "3.4.0"}
   "cudf"
   "re" {>= "1.2.0"}
   "ocamlfind" {build}

